build.sbt

```diff
@@ -24,24 +24,25 @@ dependencyOverrides += "com.typesafe.akka" %% "akka-http-core" % akkaHttpVersion
 
 val dependencies = Seq(
   ws,
-  "uk.gov.hmrc" %% "bootstrap-play-26" % "1.3.0",
-  "uk.gov.hmrc" %% "govuk-template" % "5.52.0-play-26",
+  "uk.gov.hmrc" %% "bootstrap-frontend-play-26" % "2.24.0",
+  "uk.gov.hmrc" %% "govuk-template" % "5.56.0-play-26",
   "uk.gov.hmrc" %% "play-ui" % "8.8.0-play-26",
-  "uk.gov.hmrc" %% "simple-reactivemongo" % "7.23.0-play-26",
+  "uk.gov.hmrc" %% "play-health" % "3.14.0-play-26",
+  "uk.gov.hmrc" %% "simple-reactivemongo" % "7.30.0-play-26",
   "uk.gov.hmrc" %% "play-whitelist-filter" % "3.1.0-play-26",
-  "uk.gov.hmrc" %% "totp-generator" % "0.19.0",
+  "uk.gov.hmrc" %% "totp-generator" % "0.21.0",
   "com.github.kxbmap" %% "configs" % "0.4.4",
-  "org.typelevel" %% "cats-core" % "2.0.0",
-  "org.jsoup" % "jsoup" % "1.12.2",
-  "org.mongodb.scala" %% "mongo-scala-driver" % "2.8.0"
+  "org.typelevel" %% "cats-core" % "2.1.1",
+  "org.jsoup" % "jsoup" % "1.13.1",
+  "org.mongodb.scala" %% "mongo-scala-driver" % "2.9.0"
 )
 
 def testDependencies(scope: String = "test") = Seq(
   "uk.gov.hmrc" %% "bootstrap-play-26" % "1.3.0" % scope classifier "tests",
```

Comment:
Hmm, any idea what this is about?  I can't think it will be a good idea for runtime to depend on bootstrap-frontend-play-26 but for tests to depend on bootstrap-play-26 .

Reply:
stubMessagesControllerComponents not available in bootstrap-frontend-play-26 , so here doesn't changed.

 TestSupport class is  using  uk.gov.hmrc.play.bootstrap.tools.Stubs.stubMessagesControllerComponents

Reply:
I see.  This will need fixing at some point, but I'm not over concerned.  Thanks